sarah/S3/particles.m

```diff
@@ -0,0 +1,144 @@
+
+
+ParticleDefinitions[GaugeES] = {
+      {H0,  {    PDG -> {0},
+                 Width -> 0, 
+                 Mass -> Automatic,
+                 FeynArtsNr -> 1,
+                 LaTeX -> "H^0",
+                 OutputName -> "H0" }},                         
+      
+      
+      {Hp,  {    PDG -> {0},
+                 Width -> 0, 
+                 Mass -> Automatic,
+                 FeynArtsNr -> 2,
+                 LaTeX -> "H^+",
+                 OutputName -> "Hp" }}, 
+
+      {phi13, {  PDG -> {0},
+                 Width -> 0,
+                 Mass -> Automatic,
+                 FeynArtsNr -> 1,
+                 LaTeX -> "\\phi_{1/3}",
+                 OutputName -> "phi1"}},
+
+      {phi23, {  PDG -> {0},
+                 Width -> 0,
+                 Mass -> Automatic,
+                 FeynArtsNr -> 1,
+                 LaTeX -> {"\\phi_{-2/3}","\\phi_{4/3}"},
+                 OutputName -> "phi2"}},
+      {phi43, {  PDG -> {0},
+                 Width -> 0,
+                 Mass -> Automatic,
+                 FeynArtsNr -> 1,
+                 LaTeX -> "\\phi_{4/3}",
+                 OutputName -> "phi3"}},
+
+      {VB,   { Description -> "B-Boson"}},                                                   
+      {VG,   { Description -> "Gluon"}},          
+      {VWB,  { Description -> "W-Bosons"}},          
+      {gB,   { Description -> "B-Boson Ghost"}},                                                   
+      {gG,   { Description -> "Gluon Ghost" }},          
+      {gWB,  { Description -> "W-Boson Ghost"}}
+      
+      };
+      
+      
+      
+      
+  ParticleDefinitions[EWSB] = {
+            
+      
+    {hh   ,  {  Description -> "Higgs",
+                 PDG -> {25},
+                 PDG.IX -> {101000001} }}, 
+                 
+     {Ah   ,  {  Description -> "Pseudo-Scalar Higgs",
+                 PDG -> {0},
+                 PDG.IX ->{0},
+                 Mass -> {0},
+                 Width -> {0} }},                       
+      
+      
+     {Hp,     { Description -> "Charged Higgs", 
+                 PDG -> {0},
+                 PDG.IX ->{0},
+                 Width -> {0}, 
+                 Mass -> {0},
+                 LaTeX -> {"H^+","H^-"},
+                 OutputName -> {"Hp","Hm"},
+                 ElectricCharge->1
+                 }},  
+
+      {phi13,  { Description -> "Phi 1/3",
+                 PDG -> {100},
+                 LaTeX -> {"\\phi_{1/3}","\\phi_{1/3}"},
+                 OutputName -> "phi1",
+                 ElectricCharge->1/3
+                 }},
+
+      {phi43,  { Description -> "Charged phi43",
+                 PDG -> {101},
+                 LaTeX -> {"\\phi_{4/3}","\\phi_{-4/3}"},
+                 OutputName -> {"phi2","conjphi2"},
```

Comment:
Here maybe `phi43`

Reply:
Done in 416c40c727b1fff32493afb31ace937f455320d0.